src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -151,33 +155,50 @@ def get_deployment_config(app):
     current_prod_env = whodaman()  # current_prod_end = <current-data-env>
```

Comment:
You didn't change this part, but thi should use the newer name. I deprecated whodaman. I can fix this when moving  this to utils.